We've disabled luajittex when building the binaries, so lets prevent texlive from trying to generate formats for it.
--- src/texmf-dist/web2c/fmtutil.cnf	2016-05-15 01:17:09.000000000 +0000
+++ ../fmtutil.cnf	2017-05-31 05:20:32.516130878 +0000
@@ -82,7 +82,7 @@
 # from luatex:
 luatex luatex language.def,language.dat.lua luatex.ini
 dviluatex luatex language.def,language.dat.lua dviluatex.ini
-luajittex luajittex language.def,language.dat.lua luatex.ini
+#! luajittex luajittex language.def,language.dat.lua luatex.ini
 #
 # from metafont:
 mf mf-nowin - -translate-file=cp227.tcx mf.ini
